prac_05/kivy_layout.kv

```diff
@@ -0,0 +1,21 @@
+# This is a Kv file.
+# It contains the declarations of the widgets that will be displayed.
+BoxLayout:
```

Comment:
Good layout choice. Using a vertical BoxLayout as the main container is a sensible choice for stacking widgets vertically.